cluster-autoscaler/provisioningrequest/pods/pods.go

```diff
@@ -29,9 +29,13 @@ import (
 
 const (
 	// ProvisioningRequestPodAnnotationKey is a key used to annotate pods consuming provisioning request.
-	ProvisioningRequestPodAnnotationKey = "cluster-autoscaler.kubernetes.io/consume-provisioning-request"
+	ProvisioningRequestPodAnnotationKey = "autoscaling.x-k8s.io/consume-provisioning-request"
 	// ProvisioningClassPodAnnotationKey is a key used to add annotation about Provisioning Class
-	ProvisioningClassPodAnnotationKey = "cluster-autoscaler.kubernetes.io/provisioning-class-name"
+	ProvisioningClassPodAnnotationKey = "autoscaling.x-k8s.io/provisioning-class-name"
```

Comment:
Let's move those annotations to the api package (not deprecated ones)

Reply:
Done